tools/daisctl/pkg/altinn/deployments_test.go

```diff
@@ -0,0 +1,213 @@
+package altinn
+
+import (
+	"github.com/altinn/altinn-platform/daisctl/pkg/kube"
+	"github.com/google/go-cmp/cmp"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestInitAppsData(t *testing.T) {
+	tests := []struct {
+		name               string
+		mockResponse       string
+		statusCode         int
+		initialDeployments *Deployments
+		expectedResult     map[string]*kube.AppVersions
+		expectError        bool
+	}{
+		{
+			name:               "Valid Response from kubewrapper, no existing apps",
+			initialDeployments: newDeployments(),
+			mockResponse: `[
+							  {
```

Comment:
It seems there might be a small indentation issue with the response string.

Would it perhaps be better to consider moving this test string to a separate file, maybe in a different directory, and then reading it from there? I feel this might offer a slightly cleaner and more organised approach, but of course, it's just a suggestion! 🙂

Reply:
We can add separate json files that we read in for the tests, I followed the style from the other tests and inlined the json.
If it was large and complicated jsons I would have placed them in a separate json files, but as these are small I chose to inline them so a reader does not need to open multiple files to see the input and output of the tests.

Should we create three json files or adjust the indentation here?

Reply:
I adjusted the indentation for now. We as a team should come to an agreement on tab sizes and if we should inline or create separate json files. I have no strong feelings towards either, but we should agree as a team :)

Reply:
I would prefer JSON file for the first two, and for the last one, my only concern is the indentation as it seems too short to have its own JSON file! 🙂

Reply:
In some of my previous go projects I have used the test name as the filename used in the input. So if we decide to move the mockResponse json into separate files I think we should do it for all of them as to not use multiple sources for the testdata. Using the name of the test as the filename for the testdata makes for easier debugging of test failures I think.
1. Test fail (suspecting something with testdata)
2. Note the name of the test
3. locate the file with the corresponding name in the testdata folder

Reply:
I'm leaving up the resolution to both of you on this PR, but I agree on this:
> Using the name of the test as the filename for the testdata makes for easier debugging 
Just consider that when you move the tests to a different file, as the error message from the test failure must be clear.

Reply:
> Just consider this when you move the tests to a different file, as the error message from the test failure must be clear.

As we use test tables i have added a folder for each group of tests and then a file with the name in that folder.
If the test fails I print the diff and the location of the testdata used as input.
I have also done the same for environments_test so we follow the same pattern